services/merak-topo/handler/topo_deploy.go

```diff
@@ -229,10 +228,10 @@ func Topo_deploy(k8client *kubernetes.Clientset, aca_image string, ovs_image str
 							ImagePullPolicy: "IfNotPresent",
 							Command:         []string{"/bin/sh", "-c", "/merak-bin/merak-agent " + aca_parameters},
 							SecurityContext: &sc,
-							Ports: []corev1.ContainerPort{
-								{Name: "gRPC", ContainerPort: 40054},
-								{Name: "prometheus", ContainerPort: 9001},
-							},
+							// Ports: []corev1.ContainerPort{
+							// 	{Name: "gRPC", ContainerPort: 40054},
+							// 	{Name: "prometheus", ContainerPort: 9001},
```

Comment:
I've fixed the deployment issue for vhost in the main branch. Please revert this change and rebase with main.

Reply:
Thanks for this kind reminder, btw, could you please do notice me when submitting any commits refer to Merak-topo next time? That would make the collaboration much more efficient and effective.

Reply:
I've added you as a code owner for merak-topo awhile back.
https://github.com/futurewei-cloud/merak/blob/main/.github/CODEOWNERS
So you should get notifications for PR review whenever it's modified.
Though it looks like you're missing some permissions on the repo.

Reply:
In case it happens again, plz ping me a msg then next time.

Reply:
Sure, I can ping you myself if I change something. However, this is not a general solution, as we may get other contributors in the future. The best thing to do would be to fix your permissions such that any contribution to merak-topo will have you auto-requested as a reviewer.

Reply:
Thanks! Conflict addressed. All extra kind ping/msg are greatly appreciated for a closer collaboration. 
Current PR includes functions for topo_id with namespace and new logging, free to test with merak-topo:dev, any issue plz free to ping me then.

Reply:
Agreed with @phudtran. Multiple contributors may work for different orgs for the same open-source projects so setting CODEOWNERS would be the solution by default. 

@phudtran which kinds of permission is needed? I saw Zeng has been added as one of contributors.

Reply:
@xieus Sure, will follow this way, thanks

Reply:
> Agreed with @phudtran. Multiple contributors may work for different orgs for the same open-source projects so setting CODEOWNERS would be the solution by default.
> 
> @phudtran which kinds of permission is needed? I saw Zeng has been added as one of contributors.

According to this file both Yan and Zhen are missing write access to the repo
https://github.com/futurewei-cloud/merak/blob/main/.github/CODEOWNERS